### PR TITLE
add glog flush to write the output to a file

### PR DESCRIPTION
**Pull request by patelpayal**
```kube-bench master  --logtostderr=false   -v 9```

 ```cat /tmp/kube-bench.INFO
Log file created at: 2019/06/28 00:01:03
Running on machine: kubemaster1
Binary: Built with gc go1.11.11 for linux/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
```

It does not log anything else after that. 

After the fix we can see the output being recorded to the file: 

```cat /tmp/kube-bench.INFO
Log file created at: 2019/06/28 00:02:27
Running on machine: kubemaster1
Binary: Built with gc go1.12 for linux/amd64
Log line format: [IWEF]mmdd hh:mm:ss.uuuuuu threadid file:line] msg
I0628 00:02:27.340695 3203197 util.go:122] Looking for config for version 1.13
I0628 00:02:27.340919 3203197 util.go:127] Looking for config file: cfg/1.13/master.yaml
I0628 00:02:27.340960 3203197 util.go:127] Looking for config file: cfg/1.12/master.yaml
...
...
```

As per https://github.com/golang/glog/blob/master/glog.go#L878 

`const flushInterval = 30 * time.Second` is the flush interval. 

https://github.com/golang/glog/blob/master/glog.go#L35-L36
```
// Log output is buffered and written periodically using Flush. Programs
// should call Flush before exiting to guarantee all log output is written
```

kube-bench exists before 30 seconds and it could not flush the output to a log directory.

### CODE REVIEW

**Comment by patelpayal** [comment]
@lizrice @mrueg - Please review.

File: cmd/root.go [hunk 1]
```diff
@@ -76,6 +76,8 @@ func Execute() {
 		fmt.Println(err)
 		os.Exit(-1)
 	}
+	// flush after exit
```
Comment (lizrice) [inline cmd/root.go]: Nit - I think you mean flush *before* exit in the comment

File: cmd/root.go [hunk 2]
```diff
@@ -76,6 +76,8 @@ func Execute() {
 		fmt.Println(err)
```
Comment (lizrice) [inline cmd/root.go]: Do we not need to also flush the log here in case of an error?

File: cmd/root.go (hunk 1 above)
Comment (patelpayal) [reply cmd/root.go]: that's right. it is fixed now.

File: cmd/root.go (hunk 2 above)
Comment (patelpayal) [reply cmd/root.go]: yes, I agree, Added `glog.flush() before exit on error`.

**Comment by patelpayal** [comment]
@lizrice Thanks for merging it. I am just curious, when is your next release cycle?

**Pull request merged**